Makefile

```diff
@@ -0,0 +1,43 @@
+# Makefile for building the project
+
+app_name=firstrunwizard
+project_dir=$(CURDIR)/../$(app_name)
+build_dir=$(CURDIR)/build/artifacts
+appstore_dir=$(build_dir)/appstore
+source_dir=$(build_dir)/source
+package_name=$(app_name)
+
+all: dist
+
+clean:
+	rm -rf $(build_dir)
+
+dist: clean
+	mkdir -p $(source_dir)
+	tar cvzf $(source_dir)/$(package_name).tar.gz $(project_dir) \
+	--exclude-vcs \
+	--exclude=$(project_dir)/build/artifacts \
+	--exclude=$(project_dir)/js/node_modules \
+	--exclude=$(project_dir)/js/coverage
+
+appstore: appstore_package
```

Comment:
from my understanding appstore related stuff can be removed - this is a shipped app as part of the release and not available on the appstore